ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/V2/Forker.hs

```diff
@@ -160,19 +164,12 @@ implForkerCommit env = do
                   AS.Empty _ -> pure ()
                   _ AS.:< closeOld' -> closeLedgerSeq (LedgerSeq closeOld')
                 -- Finally, close the anchor of @lseq@ (which is a duplicate of
-                -- the head of @olddb'@).
-                --
-                -- Note if the resource registry used to create the Forker is
-                -- ephemeral as the one created on each Chain selection or each
-                -- Forging loop iteration, this first duplicated state will be
-                -- closed by the resource registry closing down, so this will be
-                -- a double release, which is fine. We prefer keeping this
-                -- action just in case some client passes a registry that
-                -- outlives the forker.
-                --
-                -- The rest of the states in the forker will be closed via
-                -- @foeResourcesToRelease@ instead of via the registry.
-                close $ tables $ AS.anchor lseq
+                -- the head of @olddb'@). To close this handle, we have to
+                -- release the 'foeInitialHandleKey' as that one is registered
+                -- on the registry used to open the forker. Releasing it will
+                -- call 'close' on the handle which will call 'release' on the key
+                -- for the handle in the 'lgrRegistry'.
```

Comment:
> Releasing it will call `close` on the handle which will call `release` on the key  for the handle in the `lgrRegistry`.

Huh, where does the `lgrRegistry` come in here?